digitaltwins/service/test/_digital_twin_service_client_test.js

```diff
@@ -236,367 +236,38 @@ describe('DigitalTwinServiceClient', function () {
     assert.deepEqual(returnedPromise._response, testUserPatch);
   });
 
-  /* Tests_SRS_NODE_DIGITAL_TWIN_SERVICE_CLIENT_12_005: [The `getComponents` method shall call the `getComponents` method of the protocol layer with the given arguments.]*/
-  /* Tests_SRS_NODE_DIGITAL_TWIN_SERVICE_CLIENT_12_006: [The `getComponents` method shall call the callback with an error parameter if a callback is passed..]*/
-  it('getComponents calls getComponents on the PL client', function (testCallback) {
+  it('invokeComponentCommand called with the right parameters and shall return a promise', async () => {
```

Comment:
>it( [](start = 2, length = 3)

No tests for invalid number of parameters?

Reply:
Implicitly there is with the CalledWith

---
In reply to: [452338541](https://github.com/Azure/azure-iot-sdk-node/pull/844#discussion_r452338541) [](ancestors = 452338541)